src/Raven.Server/Documents/Sharding/Operations/Queries/AbstractShardedQueryOperation.cs

```diff
@@ -69,6 +69,12 @@ protected static void CombineSingleShardResultProperties(QueryResult<List<TResul
         if (combinedResult.LastQueryTime < singleShardResult.LastQueryTime)
             combinedResult.LastQueryTime = singleShardResult.LastQueryTime;
 
+        if (singleShardResult.ScannedResults != null)
+        {
+            combinedResult.ScannedResults ??= 0;
+            combinedResult.ScannedResults += singleShardResult.ScannedResults;
```

Comment:
where filter is happening actually? In the orchestrator only? then it should be 0 always and the final result should get the value from orchestrator only?

Reply:
map indexes - on the shards
map-reduce - on the orchestrator (`singleShardResult.ScannedResults` will be null)